programs/solvent-protocol/src/constants.rs

```diff
@@ -8,6 +8,7 @@ pub const LIQUIDATION_REWARD_PERCENTAGE: u8 = 20;
 pub const DROPLETS_PER_NFT: u8 = 100;
 pub const REDEEM_FEE_BASIS_POINTS: u8 = 200;
 pub const SWAP_FEE_BASIS_POINTS: u8 = 50;
+pub const DISTRIBUTOR_FEE_PERCENTAGE: u8 = 10;
```

Comment:
What do you think about using basis points here instead of percentage? That way we are allowed more granularity if we need that, and also consistent with the other constants.

Reply:
good idea